### PR TITLE
fix(pg) remove pg status check

### LINKED ISSUE #474: status check should not return error
returning error will cause the re-enqueue interval to become longer and longer

```
if pg.Status.PostgresClusterStatus != "Running" {
		return nil, errors.New("database is not ready")
	}
```

### PR DESCRIPTION
**Pull request by Ghostbaby**
remove pg status check

fix #474

Signed-off-by: zhuhuijun <zhuhuijunzhj@gmail.com>

### CODE REVIEW

File: pkg/cluster/controllers/database/readiness.go [hunk 1]
```diff
@@ -61,10 +61,6 @@ func (p *PostgreSQLController) Readiness(ctx context.Context) (*lcm.CRStatus, er
 		return nil, err
 	}
 
-	if pg.Status.PostgresClusterStatus != "Running" {
-		return nil, errors.New("database is not ready")
-	}
-
 	p.Log.Info("Database already ready.",
```
Comment (steven-zou) [inline pkg/cluster/controllers/database/readiness.go:64]:
```suggestion
	p.Log.Info("Database is ready.",
```

File: pkg/cluster/controllers/database/readiness.go [hunk 2]
```diff
@@ -61,10 +61,6 @@ func (p *PostgreSQLController) Readiness(ctx context.Context) (*lcm.CRStatus, er
 		return nil, err
 	}
 
-	if pg.Status.PostgresClusterStatus != "Running" {
```
Comment (steven-zou) [inline pkg/cluster/controllers/database/readiness.go:64]: The staus checking here should not be deleted.

```
// Define this status as a const 
PsqlRunningStatus = "Running"

// --- some code

properties := &lcm.Properties{}
crStatus := lcm.New(goharborv1alpha2.DatabaseReady).WithProperties(*properties)

// Not ready yet
if pg.Status.PostgresClusterStatus != PsqlRunningStatus {
  crStatus.WithStatus(corev1.ConditionFalse).
  WithReason("database is not ready").
  WithMessage(fmt.Sprintf("psql is %s", pg.Status.PostgresClusterStatus ))

return crStatus
}

// --- some code here 
p.Log.Info("Database is ready.",
		"namespace", p.HarborCluster.Namespace,
		"name", p.HarborCluster.Name)

	if err := p.DeployComponentSecret(conn, getDatabasePasswordRefName(name)); err != nil {
		return nil, err
	}

	addProperties(name, conn, properties)

	crStatus := WithStatus(corev1.ConditionTrue).
                WithReason("database is ready").
		WithMessage("harbor component database secrets are created.").
		WithProperties(*properties)

	return crStatus, nil
```

**Review by steven-zou — CHANGES_REQUESTED** [review]
@Ghostbaby 

Please also remove the useless const values defined in this file

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by heww — APPROVED** [review]

**Pull request merged**